src/ngrx/13-create-selectors/create-selectors.md

```diff
@@ -5,4 +5,48 @@
 
 @body
 
-TODO
\ No newline at end of file
+> **Quick Start**: You can checkout [this branch](https://github.com/bitovi/angular-ngrx-chat/tree/test-reducer) to get your codebase ready to work on this part.
+
+## Overview
+
+1. Create Selector for `userId`
+
+1. Create Selector for `username`
+
+1. Create Selector for `token`
+
+## Problem 1: TODO
+
+TODO
+
+## P1: What You Need to Know
+
+TODO
+
+## P1: Solution
+
+<details>
+<summary>src/app/store/login/login.selectors.ts</summary>
+
+@diff ../12-test-reducer/login.selectors.ts ./login.selectors-user-id.ts only
+
+</details>
+
+## Problem 2: TODO
+
+TODO
+
+## P2: What You Need to Know
+
+TODO: (remove?)
```

Comment:
I think it's fine to remove if there's previous explanation

Reply:
Yea, I think I'll remove, but I'll keep it for now since we haven't added copy yet